src/PathReporter.ts

```diff
@@ -15,7 +15,7 @@ function stringify(v: any): string {
     }
     return v > 0 ? 'Infinity' : '-Infinity'
   }
-  return JSON.stringify(v)
+  return JSON.stringify(v, (_key: string, value: unknown) => (typeof value === 'bigint' ? value.toString() : value))
```

Comment:
this way, bigints look like exactly strings which may be confusing: when passing a bigint to a property that accepts base-10 encoded integer strings, it will say `Invalid value "10" supplied to prop` when the string "10" is valid but `BigInt(10)` isnt

Reply:
good point, this suggestion makes sense to me.  Or `${value}n` for short.

Reply:
Took your suggestion, pushed. thanks again